src/platform/packages/shared/kbn-scout/src/playwright/runner/run_tests.ts

```diff
@@ -133,6 +141,12 @@ export async function runTests(log: ToolingLog, options: RunTestsOptions) {
   const pwGrepTag = getPlaywrightGrepTag(options.mode);
   const pwConfigPath = options.configPath;
   const pwProject = getPlaywrightProject(options.testTarget, options.mode);
+  const globalFlags = getFlags(process.argv.slice(2), {
+    allowUnexpected: true,
+  });
+  // Temporarily use `debug` log level for Playwright tests to better understand performance issues;
+  // We are going to change it to `info` in the future. This change doesn't affect Test Servers logging.
+  const logsLevel = pickLevelFromFlags(globalFlags, { default: 'debug' });
```

Comment:
does this need to default to debug? for whom are these debug logs useful atm?

Reply:
Mostly for appex-qa to evaluate parallel testing with Kibana spaces and spot some slow API calls. If we see a repetitive API calls taking 10 seconds to setup env, we usually ask to consider improvement. The end goal is not only move to playwright, but finally start writing stable, fast and reliable tests.